manifests/c/Cryptomator/Cryptomator/1.14.2/Cryptomator.Cryptomator.installer.yaml

```diff
@@ -0,0 +1,31 @@
+# Created with WinGet Releaser v2 using Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Cryptomator.Cryptomator
+PackageVersion: 1.14.2
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.17134.0
+InstallerType: wix
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+FileExtensions:
+- c9r
+- cryptomator
+ProductCode: '{161703F0-7B1C-3944-AEE3-8B7581BA9396}'
+ReleaseDate: 2024-11-19
+AppsAndFeaturesEntries:
+- DisplayVersion: 1.14.2.5384
+  ProductCode: '{3872290D-61BB-3B70-86B7-035FF4ED0D3F}'

```

Comment:
```suggestion
```

This value is incorrect. Looks like it wasn't updated in the manifest creation process. Remove this value as it's defined with correct value at Installer level & not necessarily needed at ARP level

Reply:
cc @infeo